src/main.scss

```diff
@@ -0,0 +1,22 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: "Roboto", sans-serif;
+}
+
+// Återanvändabara klasser för större skärmar
```

Comment:
Maybe use english for comments.

Reply:
I agree, thats not my comment. But can change it.